readme.md

```diff
@@ -1,151 +1,7 @@
-
-    $ export PATH=$PATH:/home/user/.gradle/wrapper/dists/gradle-2.4-all/WRAPPER_GENERATED_HASH/gradle-2.4/bin
-
-The changes will take effect once you restart the terminal or `source ~/.bashrc`.
-
-## Tasks
-
-List all tasks runnable from root project:
-
-    $ gradle tasks
-
-### IntelliJ IDEA
-
-Generate IntelliJ IDEA project:
-
-    $ gradle idea
-
-Open with `File` -> `Open Project`.
-
-### Configure
-
-Copy [configuration-example.yaml](configuration-example.yaml) to `configuration.yaml`. Modify as necessary, being careful to avoid committing sensitive data.
-
-### Build
-
-Build the project:
-
-    $ gradle build
-
-JARs [will be saved](https://github.com/johnrengelman/shadow#using-the-default-plugin-task) into the directory `build/libs/`.
-
-### Run
-
-Run the project:
-
-    $ gradle run
-
-## Base a New Project off the Skeleton
-
-Clone the skeleton:
-
-    $ git clone --origin skeleton git@github.com:osu-mist/web-api-skeleton.git my-api
-    $ cd my-api
-
-Rename the webapiskeleton package and SkeletonApplication class:
-
-    $ git mv src/main/groovy/edu/oregonstate/mist/webapiskeleton src/main/groovy/edu/oregonstate/mist/myapi
-    $ vim src/main/groovy/edu/oregonstate/mist/myapi/SkeletonApplication.class
-
-Update gradle.properties with your package name and main class.
-
-Replace swagger.yaml with your own API specification.
-
-Update configuration-example.yaml as appropriate for your application.
-
-Update the resource examples at the end of this readme.
-
-## Base an Existing Project off the Skeleton
-
-Add the skeleton as a remote:
-
-    $ git remote add skeleton git@github.com:osu-mist/web-api-skeleton.git
-    $ git fetch skeleton
-
-Merge the skeleton into your codebase:
-
-    $ git checkout feature/abc-123-branch
-    $ git merge skeleton/master
-    ...
-    $ git commit -v
-
-
-## Incorporate Updates from the Skeleton
-
-Fetch updates from the skeleton:
-
-    $ git fetch skeleton
-
-Merge the updates into your codebase as before.
-Note that changes to CodeNarc configuration may introduce build failures.
-
-    $ git checkout feature/abc-124-branch
-    $ git merge skeleton/master
-    ...
-    $ git commit -v
-
-
-## Resources
-
-The Web API definition is contained in the [Swagger specification](swagger.yaml).
-
-The following examples demonstrate the use of `curl` to make authenticated HTTPS requests.
-
-### GET /
-
-This resource returns build and runtime information:
-
-    $ curl \
-    > --cacert doej.pem \
-    > --user "username:password" \
-    > https://localhost:8080/api/v0/
-    {"name":"web-api-skeleton","time":"2016-08-02 14:37:01-0700","unixTime":1470173821035,"commit":"e3d396e","documentation":"swagger.yaml"}
-
-NOTE: you should only specify a certificate with --cacert for local testing.
-Production servers should use a real certificate
-issued by a valid certificate authority.
+## GET /
+### idimages/{osuid}
```

Comment:
Let's specify what headers and mime-type and other information your api returns.